torch/_export/passes/replace_with_hop_pass_util.py

```diff
@@ -171,6 +171,15 @@ def _replace_with_hop_pass_helper(
             )
             setattr(new_gm, node.target, new_subgm)
 
+    # The process of inlining nodes seems to break node.users when a node is present in
```

Comment:
Instead of fixing it up here, can we try to figure out why the inline call doesn't properly change the users and fix it up there? Then we have a better inline function that's less buggy, which improves the composability of the system.